web-api/src/storage.rs

```diff
@@ -180,20 +182,31 @@ pub(crate) trait Tag {
 pub struct Config {
     elastic: elastic::Config,
     postgres: postgres::Config,
+    #[cfg(feature = "ET-3837")]
+    is_migrated: bool,
```

Comment:
In general such information would be stored in the database instead of the config.

(Through not as a bool, but more like a migration version number/state.)

Instead of then using a endpoint you could automatically start the migration in the background when loading a binary/service containing the code, use the migration state to make sure that only n (here e.g. 1) instance slowly does the migration, have handling in case the instance with the migration disappears (in the future, not for now) and just poll the state every few minutes until the migrations are done to the expected state.

As the migrations only go one way you still might use a indicator in the app state to avoid additional database queries.